csrankings.csv

```diff
@@ -20343,7 +20345,7 @@ Ying Zou,Queen's University,https://www.ece.queensu.ca/people/Y-Zou,pCYlknMAAAAJ
 Ying-Dar Lin,National Chiao Tung University,http://speed.cis.nctu.edu.tw/~ydlin,6ZzMxhsAAAAJ
 Ying-Ping Chen,National Chiao Tung University,http://www.cs.nctu.edu.tw/~ypchen,5Fb9380AAAAJ
 Ying-li Tian,CUNY,http://www-ee.ccny.cuny.edu/wwwn/yltian/home.html,aAWeB4wAAAAJ
-Yingbin Liang,Syracuse University,http://yliang06.mysite.syr.edu,lGgLAiIAAAAJ
+Yingbin Liang,Ohio State University,https://sites.google.com/view/yingbinliang,lGgLAiIAAAAJ

```

Comment:
No apparent cross-appointment (see https://www.cse.osu.edu/directory/categories/appointment/courtesy-appointments).

Reply:
I just restored this entry, can you please check again?